foxpuppet/windows/browser/window.py

```diff
@@ -24,6 +25,19 @@ class BrowserWindow(BaseWindow):
     _tab_browser_locator = (By.ID, 'tabbrowser-tabs')
 
     @property
+    def tab_bar(self):
```

Comment:
I prefer the name `tab_bar` but note this is inconsistent with `navbar`, which should probably have the underscore.